test/CoreTest.cpp

```diff
@@ -557,3 +557,24 @@ TEST_CASE( "wrapper_test", "[core]" )
     REQUIRE(pp["numParticles"][RecordComponent::SCALAR].m_chunks->empty());
 #endif
 }
+
+TEST_CASE( "use_count_test", "[core]" )
+{
+    Series o = Series("./new_openpmd_output", AccessType::CREATE);
```

Comment:
no file ending? :-o how does it know it's backend? :-o

Reply:
If no known filename extension can be matched, we can't determine a suitable backend and [throw](https://github.com/C0nsultant/libopenPMD/blob/0b97c08c0094bc52f461d15d3d905e43b7ac44a6/src/Series.cpp#L798) (when #229 is merged). With the only exception being *no file extension at all*, in which case we do [NOOPs](https://github.com/openPMD/openPMD-api/blob/63dc8818d12ba868466e950b5c252afec71536af/src/IO/AbstractIOHandler.cpp#L106) with help of [```Format::DUMMY```](https://github.com/openPMD/openPMD-api/blob/e05b136b2b6a4d6247be573fc44922f943d49ce7/src/Series.cpp#L770).